src/lib/queue/pipeline.ts

```diff
@@ -0,0 +1,443 @@
+import {
+	KeetaAnchorQueueRunner
+} from './index.js';
+import type {
+	KeetaAnchorQueueStorageDriver,
+	KeetaAnchorQueueRequestID,
+	KeetaAnchorQueueCommonOptions,
+	KeetaAnchorQueueEntry
+} from './index.ts';
+import { MethodLogger } from './internal.js';
+import type { Logger } from '../log/index.ts';
+import type { JSONSerializable } from '../utils/json.js';
+
+/**
+ * A KeetaAnchorQueueRunner that uses `any` for the user types
+ * This is helpful for expressing pipelines where the inputs
+ * and outputs of various stages are not known ahead of time
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+class KeetaAnchorQueueRunnerAny<UREQUEST = any, URESPONSE = any, REQUEST extends JSONSerializable = JSONSerializable, RESPONSE extends JSONSerializable = JSONSerializable> extends KeetaAnchorQueueRunner<UREQUEST, URESPONSE, REQUEST, RESPONSE> {
```

Comment:
Could we rename these to something like this?


```suggestion
class KeetaAnchorQueueRunnerAny<ParsedRequest = any, ParsedResponse = any, SerializedRequest extends JSONSerializable = JSONSerializable, SerializedResponse extends JSONSerializable = JSONSerializable> extends KeetaAnchorQueueRunner<ParsedRequest, ParsedResponse, SerializedRequest, SerializedResponse> {
```